src/AppBundle/Repository/MatchingContextRepository.php

```diff
@@ -37,6 +38,8 @@ public function createQueryForPublicMatchingContexts($matchingContextAlias = 'mc
             ->select($matchingContextAlias)
             ->leftJoin("$matchingContextAlias.notice", $noticeAlias)
             ->leftJoin("$noticeAlias.contributor", $contributorAlias)
+            ->leftJoin('AppBundle:Relay', 'r', Join::WITH, "r.notice = $noticeAlias.id")
+            ->leftJoin('r.contributor', 'rc')
```

Comment:
`relayedBy`

Reply:
Yes, I pushed too fast but the tests got by back :sweat_smile: 

Fixed !